skimage/draw/_random_shapes.py

```diff
@@ -166,7 +172,7 @@ def _generate_triangle_mask(point, image, shape, random):
 # Allows lookup by key as well as random selection.
 SHAPE_GENERATORS = dict(
     rectangle=_generate_rectangle_mask,
-    circle=_generate_circle_mask,
+    circle=_generate_disk_mask,
```

Comment:
```suggestion
    disk=_generate_disk_mask,
```
I suggest to rename this as well